application/src/main/java/org/togetherjava/tjbot/features/moderation/scam/ScamBlocker.java

```diff
@@ -232,10 +232,12 @@ private void reportScamMessage(MessageReceivedEvent event, String reportTitle,
         }
 
         User author = event.getAuthor();
+        String avatarOrDefaultUrl = author.getEffectiveAvatarUrl();
```

Comment:
the var seems unnecessary

Reply:
I disagree, there's three kinds of urls method. `default`, `effective` and `avatar`. Now that you know what `effective` means, it prolly seems obvious but it's not. In the long run `avatarOrDefault` provides more accurate information imo. I actually had to test that `effective` meant `avatarOrDefault`.

Reply:
just see the docs?

Reply:
![image](https://github.com/Together-Java/TJ-Bot/assets/61616007/2208b64b-6bca-4d21-82ee-2d50d34fdfda)

I have and i still think the variable is better.